### PR TITLE
Resistor color

### PR DESCRIPTION
**Pull request by annaoliinyk**
Instructions
If you want to build something using a Raspberry Pi, you'll probably use resistors. For this exercise, you need to know two things about them:

Each resistor has a resistance value.
Resistors are small - so small in fact that if you printed the resistance value on them, it would be hard to read.
To get around this problem, manufacturers print color-coded bands onto the resistors to denote their resistance values. Each band has a position and a numeric value.

The first 2 bands of a resistor have a simple encoding scheme: each color maps to a single number.

In this exercise you are going to create a helpful program so that you don't have to remember the values of the bands.

These colors are encoded as follows:

Black: 0
Brown: 1
Red: 2
Orange: 3
Yellow: 4
Green: 5
Blue: 6
Violet: 7
Grey: 8
White: 9
The goal of this exercise is to create a way:

to look up the numerical value associated with a particular color band
to list the different band colors

### CODE REVIEW

File: resistor_color/resistor_color_task.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "green": 5,
+        "blue": 6,
+        "violet": 7,
+        "grey": 8,
+        "white": 9
+    }
+    return switch.get(color)
+
+
+def colors():
```
Comment (AnaKovaGH) [inline resistor_color/resistor_color_task.py]: What is the purpose of this function? Should it be the constant?

File: resistor_color/resistor_color_task.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        "black": 0,
+        "brown": 1,
+        "red": 2,
+        "orange": 3,
+        "yellow": 4,
+        "green": 5,
+        "blue": 6,
+        "violet": 7,
+        "grey": 8,
+        "white": 9
```
Comment (AnaKovaGH) [inline resistor_color/resistor_color_task.py]: Dict should be in a separate constant outside of the function

Comment (annaoliinyk) [reply resistor_color/resistor_color_task.py]: This was fixed, changes pushed

File: resistor_color/resistor_color_task.py (hunk 1 above)
Comment (annaoliinyk) [reply resistor_color/resistor_color_task.py]: Per exercise, that should be a function. Added colors_list as a constant.
Below is a test from the website:
expected = [
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white",
]
self.assertEqual(colors(), expected)

File: resistor_color/resistor_color_task.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Task from https://exercism.org/tracks/python/exercises/resistor-color
+
+switch = {
```
Comment (AnaKovaGH) [inline resistor_color/resistor_color_task.py]: Constants should be written in UPPER CASE

File: resistor_color/resistor_color_task.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    "orange": 3,
+    "yellow": 4,
+    "green": 5,
+    "blue": 6,
+    "violet": 7,
+    "grey": 8,
+    "white": 9
+}
+
+colors_list = list(switch.keys())
```
Comment (AnaKovaGH) [inline resistor_color/resistor_color_task.py]: Same here

Comment (AnaKovaGH) [inline resistor_color/resistor_color_task.py]: Please remove not used code

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change what was mentioned

**Comment by annaoliinyk** [comment]
This was fixed

File: resistor_color/resistor_color_task.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+def color_code(color: str):
+    try:
+        return SWITCH.get(color)
+    except:
+        logging.error("Invalid input")
+
+
+# method to list all the existing colors
+def colors():
```
Comment (AnaKovaGH) [inline resistor_color/resistor_color_task.py]: Function name should be an action, not just a noun. 
Like, "get_colors()"

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Please change the function naming

**Comment by annaoliinyk** [comment]
This was done

**Review by AnaKovaGH — APPROVED** [review]

**Pull request merged**